modules/orca/main.tf

```diff
@@ -122,6 +122,8 @@ module "orca_secretsmanager" {
   db_user_password  = var.db_user_password
   db_host_endpoint  = var.db_host_endpoint
   restore_object_role_arn = module.orca_iam.restore_object_role_arn
+  s3_access_key = ""
+  s3_secret_key = ""
```

Comment:
what are the functions of these two var?

Reply:
They need to be populated in a future task. For now, dummy values allow us to build and deploy.